mindmeld/text_preparation/normalizers.py

```diff
@@ -195,28 +196,105 @@ def normalize(self, text):
         return unicodedata.normalize(self.normalization_type, text)
 
 
+class Lowercase(Normalizer):
+    """Lowercase Normalizer Class."""
+
+    def normalize(self, text):
+        """
+        Args:
+            text (str): Input text.
+        Returns:
+            normalized_text (str): Normalized Text.
+        """
+        return text.lower()
+
+
+class RegexNormalizer(Normalizer):
+    """Normalization class that substitutes regex matches with either an empty string or a
+    replacement string."""
+
+    def __init__(self, regex_norm_rules):
+        """Creates a Regex Normalizer instance.
+
+        Args:
+            regex_norm_rules (List[Dict]): List of regex normalization rules represented as
+                dictionaries. The example rule below removes any text in parentheses.
+                {
+                    "description": "remove_text_in_parantheses",
+                    "pattern": "\(.+?\)",
+                    "replacement": ""
+                }
+        """
+        super().__init__()
+        self.regex_norm_rules = regex_norm_rules
+
+    def normalize(self, text):
+        """Apply Regex substitutions to the given text.
+        Args:
+            text (str): Input text.
+        Returns:
+            normalized_text (str): Normalized Text.
+        """
+        # TODO: Handle case if the "pattern" or "replacement" is none
+
+        for regex_norm_rule in self.regex_norm_rules:
+            pattern = regex_norm_rule.get("pattern")
+            replacement = regex_norm_rule.get("replacement")
+            text = re.sub(pattern, replacement, text)
+        return text
+
+
+class MindMeldRegexNormalizer(RegexNormalizer):
```

Comment:
What is this class meant to do that `RegexNormalizer` doesn't already. I feel like we don't need this.

Reply:
Makes sense, removed. The rules in `MindMeldRegexNormalizer` are now the default so we no longer need the class to specify them